test/test_custom_ops.py

```diff
@@ -678,6 +680,134 @@ def foo(x: Tensor, y: int) -> Tensor:
 
             infer_schema(foo, mutated_args={"y"})
 
+    def test_blackbox_basic(self):
+        @def_blackbox(mutated_args=())
+        def f(x: Tensor, y: float) -> Tensor:
+            return x + y
+
+        x = torch.randn(3)
+        y = 3.14
+        z = f(x, y)
+        self.assertEqual(z, x + y)
+
+        @f.impl("cpu")
+        def _(x, y):
+            return x - y
```

Comment:
And it's weird to have implementations that give numerically different results, that's not a thing you want people to do in reality.